include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2919,11 +2920,11 @@ inline FEEvaluationBase<dim, n_components_, Number, is_face>::FEEvaluationBase(
   cell(numbers::invalid_unsigned_int),
   is_interior_face(is_interior_face),
   dof_access_index(
-    is_face ?
-      (is_interior_face ?
-         internal::MatrixFreeFunctions::DoFInfo::dof_access_face_interior :
-         internal::MatrixFreeFunctions::DoFInfo::dof_access_face_exterior) :
-      internal::MatrixFreeFunctions::DoFInfo::dof_access_cell),
+    is_face ? (is_interior_face ? internal::MatrixFreeFunctions::DoFInfo<
+                                    Number>::dof_access_face_interior :
```

Comment:
That is a strange place to break the line. Is this `clang-format`'s fault?

Reply:
This is clang-format.